mypy/stubtest.py

```diff
@@ -1040,9 +1051,28 @@ def test_stubs(args: argparse.Namespace) -> int:
     options = Options()
     options.incremental = False
     options.custom_typeshed_dir = args.custom_typeshed_dir
+    options.config_file = args.config_file
+
+    plugins = None # type: Optional[Sequence[Plugin]]
+    if options.config_file:
+        def set_strict_flags() -> None:  # not needed yet
+            return
+        parse_config_file(options, set_strict_flags, options.config_file, sys.stdout, sys.stderr)
+        errors = Errors(options.show_error_context,
```

Comment:
Since we don't ever set any of these options, the defaults match the defaults of Options, and because pretty is False we never use read_source, maybe we can just pass in `Errors()`?
That way you also no longer need to worry about cached_read etc not existing